worlds/World 2/world.mt

```diff
@@ -1 +1,4 @@
-gameid = default
\ No newline at end of file
+gameid = default
```

Comment:
These changes to 'World 1-4' are strange, i don't know how they happened, they somehow happened due to me testing mgv7p. I will of course remove these changes before any merge.